cluster-provision/gocli/cmd/run.go

```diff
@@ -812,7 +812,7 @@ func provisionK8sOptions(sshClient libssh.Client, k8sClient k8s.K8sDynamicClient
 	}
 
 	if n.CNAO {
-		cnaoOpt := cnao.NewCnaoOpt(k8sClient, sshClient)
+		cnaoOpt := cnao.NewCnaoOpt(k8sClient, sshClient, n.Multus)
```

Comment:
CNAO should also deploy the dynamic-network-controller for kubevirt/kubevirt e2e tests. 
Please ensure the current behavior remains.

Reply:
this parameter is controlled by the overall `KUBEVIRT_WITH_MULTUS_V3` param
from a callers perspective everything will work the same.
and what it previously was is that if this variable is passed multus is deployed separately. not by CNAO.
see this comment i left for or shoval inquiring about the same thing: https://github.com/kubevirt/kubevirtci/commit/7b8b9e8d9a3566a5157d742376627d55beee4255#r146222431

of if you're referring to something else let me know